docs/debugging/pydev.rst

```diff
@@ -111,7 +111,7 @@ Running/Debugging Pyramid under Pydev
    ``$WORKSPACE`` is the name of the PyDev workspace containing your project
    
    To create a working example, copy the `pyramid tutorial step03 
-   <https://pyramid_tutorials.readthedocs.org/en/latest/getting_started/03-config/index.html>`_
+   <https://pyramid_tutorials.readthedocs.io/en/latest/getting_started/03-config/index.html>`_
```

Comment:
There are some redirects, too, so just clicking the link is not enough.

FYI, I started using `linkcheck` in Sphinx in Pyramid, but I know that running it on the Cookbook will end up with a week's worth of work to clean up. /me sweeps it back under the carpet.

More info:
https://github.com/Pylons/pyramid/blob/1.7-branch/CHANGES.txt#L52-L55